skycultures/world/description.en.utf8

```diff
@@ -51,24 +51,42 @@ thus, was common in the Near East, in Europe and its colonies on all continents.
 However, between the ancient Greek constellations there were gaps of areas with only faint stars belonging to no constellation.[#47] 
 In the Modern Epoch, several astronomers suggested constellation figures for these gaps 
 (e.g. the constellation of "the lynx" for an area close to the Great Bear but with stars so faint that you would need the eyes of a lynx to see them), 
-among them many political constellations to the honor of certain kings or patrons. In the beginning of the 20th century, the IAU aimed for a huge clean-up.</p>
-<p>In 1922, the IAU officially accepted the list of 88 constellation names and their official abbreviations with three letters. 
-The only remaining political constellations is Scutum, the shield of a Polish king but without mentioning the political reference anymore. 
+among them several political constellations to the honor of certain kings or patrons. 
+In the late 19th century, the astronomical research community aimed for a huge clean-up to avoid ambiguities in assigning stars to constellations.</p>
+<p>In 1922, the IAU officially approved the list of 88 constellation names and their official abbreviations with three letters. 
+The only remaining political constellation is Scutum, the shield of a Polish king but without mentioning the political reference anymore. 
 The Belgian priest E. Delporte got the task to define constellation boundaries according to coordinates. 
-In 1928, he finished the work and the IAU accepted his delimitations [#40].</p>
+In 1928, he finished the work, based on an atlas from Argentina [#51], and the IAU accepted his delimitations [#40].</p>
 <p>These "constellations" are naked areas in the sky without any stick figures or depictions [#44]. 
-The borders simply follow the lines of right ascension and declination. 
+The borders simply follow the lines of right ascension and declination from the epoch 1875.0. 
+Precession causes a slowly increasing deviation from the originally rectilinear grid.
 These constellations became the standard way to describe the sky, replacing similar sets in other sky cultures exhaustively in daily usage.</p>
 <p>Stellarium by default displays the precise RA-DEC-boundaries and a set of artwork constellations on the basis of Ptolemaic figures but following our contemporary taste of art.</p>
 
+<h3>Asterisms</h3>
+The term "asterism" (assembly of stars) can sometimes be used synonymously to "constellation". In the modern use it describes a figure which is not part of the official canon. 
```

Comment:
Thanks for additional, but please define paragraphs (<p></p>) for text of these additionals

Reply:
Oops, yes!